packages/server/src/main/java/software/uncharted/terarium/hmiserver/controller/dataservice/SummaryController.java

```diff
@@ -51,9 +51,9 @@ public class SummaryController {
 
 	final CurrentUserService currentUserService;
 
-	@GetMapping
+	@PostMapping("/search")
```

Comment:
How is that a search, this does not accept any query?

Reply:
Its taking in a list of IDs to search for in the body. @mwdchang and I landed on `search` for the operation